### PR TITLE
Add frontend parameter for passing in country code

### PR DESCRIPTION
**Pull request by andrewmarriott-aws**
*Issue #, if available:*
#76 

*Description of changes:*
- Dropdown added in Step 2 of web app
- Country value gets passed to Glue script where corresponding normalization patterns are used

*Implementation:*
- Error if value not selected
Screenshot 2023-02-05 at 5 32 30 PM

- Can iteratively add options as more countries are supported 
Screenshot 2023-02-05 at 5 34 03 PM

- Additional information modal next to param (text updated: https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/eed6ec5ed6bfc8557c1fa4d809149c1ba66b072e)
Screenshot 2023-02-05 at 5 35 28 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/glue/amc_transformations.py [hunk 1]
```diff
@@ -104,6 +92,8 @@
         print("ERROR: Invalid user-defined value for dataset period:")
         print(user_defined_partition_size)
         exit(1)
+if 'country_code' in args:
+    country_code = args['country_code']
```
Comment (ianwow) [inline source/glue/amc_transformations.py:96]: else:
    print("Missing required arg: country_code")

**Review by ianwow — CHANGES_REQUESTED** [review]
Print and error in amc_transfomrations.py if it is invoked without the country_code parameter. Also, add that parameter into the comments (line 18 and 35)

File: source/glue/amc_transformations.py (hunk 1 above)
Comment (andrewmarriott-aws) [reply source/glue/amc_transformations.py:97]: fixed: [d106f81](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/d106f81cd66d51c36a5504dc57c793cd0468abdd) and [78d9514](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/78d95149ca8b173c597bb89562884700bafae0f8)

**Comment by andrewmarriott-aws** [comment]
> Print and error in amc_transfomrations.py if it is invoked without the country_code parameter. Also, add that parameter into the comments (line 18 and 35)

fixed: [d106f81](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/d106f81cd66d51c36a5504dc57c793cd0468abdd) and [78d9514](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/pull/83/commits/78d95149ca8b173c597bb89562884700bafae0f8)

**Comment by andrewmarriott-aws** [comment]
Fixed merge conflicts

**Review by ianwow — CHANGES_REQUESTED** [review]
resolve `npm run lint` issues

**Review by ianwow — APPROVED** [review]

**Pull request merged**